elements/lisk-p2p/src/peer.ts

```diff
@@ -151,20 +163,29 @@ export class Peer extends EventEmitter {
 		packet: unknown,
 	) => void;
 	private readonly _handleInboundSocketError: (error: Error) => void;
-	private readonly _handleInboundSocketClose: (code: number, reason: string) => void;
+	private readonly _handleInboundSocketClose: (
+		code: number,
+		reason: string,
+	) => void;
 
 	public constructor(
 		peerInfo: P2PDiscoveredPeerInfo,
 		peerConfig?: PeerConfig,
 		peerSockets?: PeerSockets,
 	) {
 		super();
+		this._reputation = DEFAULT_REPUTATION_SCORE;
 		this._peerInfo = peerInfo;
 		this._peerConfig = peerConfig ? peerConfig : {};
 		this._ipAddress = peerInfo.ipAddress;
 		this._wsPort = peerInfo.wsPort;
 		this._id = constructPeerId(this._ipAddress, this._wsPort);
 		this._height = peerInfo.height ? peerInfo.height : 0;
+		this._callCounter = 0;
+		this._counterResetInterval = setInterval(
+			() => (this._callCounter = 0),
```

Comment:
isn't it suppose to be
`() => { this._callCounter = 0; }` or it's another syntax?

Reply:
Yes they achieve the same result.